### PR TITLE
[Tempest] Install extra RPMs

### PR DESCRIPTION
**Pull request by lpiwowar**
This patch introduces tempestRun.extraRPMs parameter in the Tempest CR that allows to specifying a list of extra RPMs that should be installed before the execution of Tempest.

This parameter accepts a list of URLs that point to the RPMs that should be installed. The RPMs are installed in the order in which they are listed via the parameter.

Depends-On: https://github.com/openstack-k8s-operators/tcib/pull/175

### CODE REVIEW

**Comment by lpiwowar** [comment]
This can be merged once this PR is merged -> https://github.com/openstack-k8s-operators/tcib/pull/175

**Comment by kopecmartin** [comment]
looks awesome! add please also some doc and a warning it can't be used together with the extra plugins opt

**Review by kopecmartin — APPROVED** [review]
lgtm

**Pull request merged**
